mesonbuild/coredata.py

```diff
@@ -117,6 +117,22 @@ def toint(self, valuestring):
     def validate_value(self, value):
         return self.toint(value)
 
+class UserUmaskOption(UserIntegerOption):
+    def __init__(self, name, description, value, yielding=None):
+        super().__init__(name, description, 0, 0o777, value, yielding)
+
+    def set_value(self, newvalue):
+        if newvalue is None or newvalue == 'none':
+            self.value = None
+        else:
+            super().set_value(newvalue)
+
+    def toint(self, valuestring):
+        try:
+            return int(valuestring, 8)
+        except:
```

Comment:
__[Flake8]__

[E722] do not use bare except'


<!-- Comment by [SideCI](https://sideci.com) -->

Reply:
Fixed.

Please note that `UserIntegerOption.to_int()` a few lines above has this same problem...

(Also, I did run flake8, not sure why it doesn't complain about it in my local copy... Maybe my config is different?)